### ISSUE TITLE
Multi-Lingual

### ISSUE THREAD

**Issue by dillorscroft** [agebase/umbraco-template#1]
Wondered if you'd like to include multi-lingual samples?

Possibly via Vorto or separate sites...

**Comment by danlister** [comment]
Good suggestion @dillorscroft. I'll add it to the features list

**Comment by dillorscroft** [comment]
Separate sites might be best so that there are less dependencies.

**Comment by danlister** [comment]
Yeah that would be cool. I've seen it implemented as follows before. Seems to work quite well

- GB
  - Home
- BE
  - DE
    - Home
  - FR
    - Home
